packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -94,46 +95,6 @@ BuildRequires: python-setuptools
 %description
 A set of commands and yum plugins that support a Katello host including faster package profile uploading and bound repository reporting.  This is required for errata and package applicability reporting.
 
-%if %{build_agent}
-%package -n katello-agent
```

Comment:
Do we need to Obsolete this package so it gets removed?

Reply:
If we really want to get rid of it, I can add this. Please decide.

Reply:
Done. Thanks!